enhancements/console/dynamic-plugins.md

```diff
@@ -149,16 +149,27 @@ spec:
 
 In case the plugin needs to communicate with some in-cluster service, it can
 declare a service proxy in its `ConsolePlugin` resource using the
-`spec.proxy.services` array field. A service `name`, `namespace` and `port`
-needs to be specified.
+`spec.proxy.services` map. The key of the map is the proxy alias
+of the service and value is an object in which service `name`, `namespace` and `port`
+needs to be specified and to which the request will be proxied.
+
+```yaml
+spec:
+  proxy:
+    services:
+      <proxy-alias>:
+        name: <service-name>
+        namespace: <service-namespace>
+        port: <service-port>
+```
 
 Console backend exposes following endpoint in order to proxy the communication
 between plugin and the service:
-`/api/proxy/namespace/<service-namespace>/service/<service-name>:<port-number>/<request-path>?<optional-query-parameters>`
+`/api/proxy/namespace/<plugin-name>/<proxy-alias>/<request-path>?<optional-query-parameters>`
```

Comment:
Remove namespace from the URL?

I'd give it another path element so we can add something else under proxy/ if we want later.

```suggestion
`/api/proxy/plugin/<plugin-name>/<proxy-alias>/<request-path>?<optional-query-parameters>`
```

Reply:
sorry wanted to remove the `namespace`